core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -1447,6 +1446,19 @@ boolean scheduleTask(Workflow workflow, List<Task> tasks) {
             // rollbackTasks(workflow.getWorkflowId(), createdTasks);
             throw new TerminateWorkflowException(errorMsg);
         }
+
+        // On addTaskToQueue failures, ignore the exceptions and let WorkflowRepairService take care of republishing the messages to the queue.
```

Comment:
Does this also account for partial failures?

Reply:
Yes. Partial failures when adding to queue are fine. For eg., if only 2 of 5 messages are added to the queue and the rest failed, the sweeper service would still check if the queue contains each message as long as the task is in scheduled state, and this can be recovered.